docs/content/v0.20/architecture.md

```diff
@@ -1,36 +1,479 @@
 # KubeStellar Architecture
 
-**NOTE**: This is only a very rudimentary start.
+KubeStellar provides multi-cluster deployment of Kubernetes objects, controlled by a simple 
```

Comment:
deployment and configuration ?  Or is that implied by deployment ?

Reply:
I think it is implied by deployment, since we are talking about Kubernetes objects that can be anything (workloads, operators, policies etc.)